### PR TITLE
Reset to subtitles off when subtitle playlist errors

### PR DESCRIPTION
**Pull request by gesinger**

### CODE REVIEW

File: src/master-playlist-controller.js [hunk 1]
```diff
@@ -455,6 +455,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
       videojs.log.warn('Problem encountered with the current subtitle track. Switching' +
                        ' back to default.');
       this.subtitleSegmentLoader_.abort();
+      this.subtitlePlaylistLoader_ = null;
```
Comment (mjneil) [inline src/master-playlist-controller.js]: Why was this added here?

Comment (gesinger) [reply src/master-playlist-controller.js]: I was trying to match alt audio, but I think you're right, it isn't needed here.

File: src/master-playlist-controller.js [hunk 2]
```diff
@@ -454,8 +454,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.subtitleSegmentLoader_.on('error', () => {
       videojs.log.warn('Problem encountered with the current subtitle track. Switching' +
                        ' back to default.');
-      this.subtitleSegmentLoader_.abort();
-      this.setupSubtitles();
+      this.handleSubtitleError_();
```
Comment (mjneil) [inline src/master-playlist-controller.js]: error message is the same in both handlers, probably best to move the warn message into the callback and not use an anon func
